internal/runtime/client/client.go

```diff
@@ -254,25 +254,25 @@ func (c *client) CallAllExtensions(ctx context.Context, hook runtimecatalog.Hook
 
 // aggregateSuccessfulResponses aggregates all successful responses into a single response.
 func aggregateSuccessfulResponses(aggregatedResponse runtimehooksv1.ResponseObject, responses []runtimehooksv1.ResponseObject) {
-	// At this point the Status should always be ResponseStatusSuccess and the Message should be empty.
-	// So let's set those values to avoid keeping values that could have been set by the caller of CallAllExtensions.
-	aggregatedResponse.SetMessage("")
+	// At this point the Status should always be ResponseStatusSuccess.
 	aggregatedResponse.SetStatus(runtimehooksv1.ResponseStatusSuccess)
 
-	aggregatedRetryResponse, ok := aggregatedResponse.(runtimehooksv1.RetryResponseObject)
-	if !ok {
-		// If the aggregated response is not a RetryResponseObject then we're done.
-		return
-	}
```

Comment:
I think this was added for safety reasons. For now we control where `CallAllExtensions` are called and with what arguments. But this was added for safety when client will be imported and used by providers that might start implementing their own hooks.

Reply:
should be fixed now